ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -853,3 +853,14 @@ wan_virtual_topologies:
           - names: [internet]
             preference: alternate
 ```
+
+### WAN Validation
+
+`eos_validate_state` is being enriched to support new tests for WAN designs. The tests listed below are validating WAN designs.
+
+| AVD Test Class | ANTA Test Class | Description |
+| -------------- | --------------- | ----------- |
+| AvdTestInterfacesState | VerifyInterfacesStatus | Validate the DPS interface status. |
+| AvdTestBGP | VerifyBGPSpecificPeers | Validate the different BGP Address Families Identifiers (AFI) and Subsequent Address Family Identifiers (SAFI) sessions state. |
```

Comment:
Can you add which address families are checked for AutoVPN (path-selection) and CV Pathfinder (link-state / srte) + any other we check

Reply:
No, not right now apart from path-selection, link-state and sr-te